packages/ketcher-react/src/script/editor/Editor.ts

```diff
@@ -200,6 +206,11 @@ class Editor implements KetcherEditor {
 
     const tool = new toolMap[name](this, opts)
 
+    // hide icon if not AtomToll chosen
```

Comment:
Please, fix a typo `Toll` :)

Reply:
But this kind of comments we can replace with:
```
const isAtomToolChosen = name === 'atom'
if (!isAtomToolChosen) {
  ...
}
```
I think this makes the same sense.